docs/source/develop.rst

```diff
@@ -86,23 +86,19 @@ Contributions to Dask can then be made by submitting pull requests on GitHub.
 Install
 ~~~~~~~
 
-You may want to install larger dependencies like NumPy and Pandas using a
-binary package manager like conda_.  You can skip this step if you already
-have these libraries, don't care to use them, or have sufficient build
-environment on your computer to compile them when installing with ``pip``::
+To build the library you can install the necessary requirements using
+``pip`` or ``conda``::
 
-   conda install -y numpy pandas scipy bokeh psutil
+  cd dask
 
-.. _conda: https://conda.io/
-
-Install Dask and dependencies::
+``pip``::
 
-   cd dask
-   python -m pip install -e ".[complete]"
+  python -m pip install -e ".[complete]"
 
-For development, Dask uses the following additional dependencies::
+``conda``::
 
-   python -m pip install pytest moto
+  conda env create -f continuous_integration/environment-3.8-dev.yaml
```

Comment:
I usually use this method, however we will have to keep this reference to the latest Python version up-to-date if we explicitly call it out in the docs. Perhaps the Python version should be removed from the env file name and just say `latest`?

You may also want to add `--name=dask-dev` to give the conda environment a more descriptive name.

Reply:
I think git can store symlinks? Maybe add a symlink to the folder, which we can update when new environments are added?

Reply:
Is it worth creating a new yaml file called `environment-latest-dev.yaml? i.e. can specify dask-dev in the name https://github.com/dask/dask/blob/master/continuous_integration/environment-3.8-dev.yaml#L2

Any idea how to setup the latest python version in the yaml? https://github.com/dask/dask/blob/master/continuous_integration/environment-3.8-dev.yaml#L7
Or keep it at 3.8.* until it eventually switches to 3.9.*

Reply:
Using one of our test environments we use as part of CI is the best way to guarantee that a developer's system matches our test system. I'd rather not create a new file that could get out of date - using one that we use for CI is the easiest for us to maintain.

Users can always override the name of the environment with a `-n` flag on `conda env create`, so having the name be `dask-dev` in the environment file doesn't really matter.

```
  conda env create -n dask-dev -f continuous_integration/environment-3.8-dev.yaml
```

I missed earlier though, we probably don't want to use `environment-3.8-dev.yaml` though, since that's for testing against upstream versions  of pandas/numpy. Rather we should use `environment-3.8.yaml`.

To keep the change minimal, could you add a symlink to that file at `environment-latest.yaml`? We can bump the symlink when we increment our CI versions.

Reply:
+1 for symlink